java/Cherepnin_Oleh/src/com/zoolatech/lecture4/tasks/_2/Tree.java

```diff
@@ -0,0 +1,88 @@
+package com.zoolatech.lecture4.tasks._2;
+
+public class Tree {
+    private Node head;
+
+    public Tree() {
```

Comment:
you have only a single constructor, so you don't need to add a default one without any implementation - it will be added by the compiler